tests/settings/SectionTest.py

```diff
@@ -1,5 +1,6 @@
-import unittest
+import logging
 import os
+import unittest
```

Comment:
would be nice if you put the relocation of `unittest` into a different commit :)